src/MoBi.Presentation/Presenter/Main/NotificationPresenter.cs

```diff
@@ -207,9 +191,24 @@ private void updateNotificationWith(IEnumerable<NotificationMessageDTO> notifica
 
       private bool notificationIsVisible(NotificationMessageDTO message)
       {
+         if (!_userSettings.ShowUnresolvedEndosomeMessagesForInitialConditions && isUnresolvedEndosomeForInitialConditionMessage(message))
```

Comment:
Should we reorganize to be a bt more readable
if(isOsberverMessagxx)
return showPKSm

if(isEndosomieError)
return sxxx

return true